cucumber/src/main/java/foundation/stack/datamill/cucumber/CommandLineSteps.java

```diff
@@ -100,13 +100,21 @@ public void move(String from, String to) throws IOException {
 
     @When("^" + Phrases.SUBJECT + " creates \"(.+)\" in (?:a|the) temporary directory with content:$")
     public void createFile(String file, String content) throws IOException {
+        doCreateFile(file, content);
+    }
+
+    @When("^" + Phrases.SUBJECT + " creates \"(.+)\" in (?:a|the) temporary directory with content as provided:$")
```

Comment:
I don't think we should add another one here - just a single step "with content:" which allows placeholders and they may optionally not use any placeholders